contracts/deploy/mainnet/147_claim_rewards_module_upgrade.js

```diff
@@ -0,0 +1,51 @@
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+const addresses = require("../../utils/addresses");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "147_claim_rewards_module_upgrade",
+    forceDeploy: false,
+    forceSkip: false,
+    reduceQueueTime: true,
+    proposalId: "",
+  },
+  async ({ deployWithConfirmation }) => {
+    const cOUSDCurveAMOProxy = await ethers.getContract("OUSDCurveAMOProxy");
+    const cOETHCurveAMOProxy = await ethers.getContract("OETHCurveAMOProxy");
+    const cGauntletUSDCStrategyProxy = await ethers.getContract(
+      "MorphoGauntletPrimeUSDCStrategyProxy"
+    );
+    const cGauntletUSDTStrategyProxy = await ethers.getContract(
+      "MorphoGauntletPrimeUSDTStrategyProxy"
+    );
+    const cMetaMorphoStrategyProxy = await ethers.getContract(
+      "MetaMorphoStrategyProxy"
+    );
+
+    await deployWithConfirmation("ClaimStrategyRewardsSafeModule", [
+      addresses.multichainStrategist,
+      // Defender Relayer
+      "0x4b91827516f79d6F6a1F292eD99671663b09169a",
```

Comment:
Maybe we can add it on the list of addresses?

Reply:
It already is `addresses.mainnet.validatorRegistrator`

Reply:
Makes sense, will use that one